guidelines/index.html

```diff
@@ -2634,17 +2634,17 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
       
       <li>
          
-         <p><strong>頭字語</strong>は、(名称や語句の中の) 頭文字あるいは他の単語の一部から作られた省略形で、一つの単語のように発音されることがあるものである。
+         <p><strong>頭字語</strong>は、(名称又は語句の中の) 頭文字又は他の単語の一部から作られた省略形で、一つの単語のように発音されるものである。
          </p>
          
-         <p class="example">NOAA は、米国の "National Oceanic and Atmospheric Administration" の頭文字による頭字語である。
+         <p class="example">NOAA は、米国の "National Oceanic and Atmospheric Administration"  の頭文字から作られた頭字語である。
          </p>
          
       </li>
       
    </ol>
    
-   <div class="note" id="issue-container-generatedID-41"><div role="heading" class="note-title marker" id="h-note-41" aria-level="3"><span>注記</span></div><p class="">頭文字語だったものを社名として採用している会社がある。そのような場合、その会社の新しい社名はその文字列であるにすぎず (例: Ecma)、その単語はもはや略語とは考えられない。
+   <div class="note" id="issue-container-generatedID-41"><div role="heading" class="note-title marker" id="h-note-41" aria-level="3"><span>注記</span></div><p class="">頭文字語だったものを社名として採用している会社がある。そのような場合、その会社の新しい社名はその文字列であるにすぎず (例: Ecma)、その単語はもはや略語とは見なされない。
```

Comment:
「みなす」はひらがな表記だったように思います。
```suggestion
   <div class="note" id="issue-container-generatedID-41"><div role="heading" class="note-title marker" id="h-note-41" aria-level="3"><span>注記</span></div><p class="">頭文字語だったものを社名として採用している会社がある。そのような場合、その会社の新しい社名はその文字列であるにすぎず (例: Ecma)、その単語はもはや略語とはみなされない。
```